### ISSUE TITLE
Cannot Send GAS out of my NEON wallet

### ISSUE THREAD

**Issue by wowology** [CityOfZion/neon-wallet#339]
Hi, I'm trying to send GAS out of my NEON wallet, version 0.0.7 and I always get 'Transaction Failed'.

Can anybody tell me how to resolve this?

Thanks!

**Comment by wowology** [comment]
I have even downloaded the wallet anew, version 0.0.7, but still am not able to send any GAS.  Is this a bug in the wallet?

If yes, what is another secure way to access my account?

Thanks for the help!

**Comment by stevesbrain** [comment]
@wowology Is it a large amount of GAS? I believe that has caused some issues, and it is being fixed in the next version

**Comment by wowology** [comment]
No , even 1 unit of GAS cannot be sent. when is next wallet out?

What can I do in between?

**Comment by stevesbrain** [comment]
@wowology I haven't had issues sending such small amounts of GAS. Does the same issue happen for you on testnet?

**Comment by wowology** [comment]
Yes, it happens all the time, cannot send GAS.

I can claim my GAS though.

Please advise

**Comment by stevesbrain** [comment]
@wowology Just to confirm, you are saying this happens all the time on mainnet, and on testnet? Have you confirmed the checksum of the downloaded file as well?

**Comment by wowology** [comment]
I am having this issue on main net.  On test net, I have 0 gas so how do I test? and if it works on testnet, how do i get my gas on main net transferred?

**Comment by wowology** [comment]
It always says 'Transaction Failed'

**Comment by stevesbrain** [comment]
@wowology It's not that you would transfer mainnet to testnet; they're two separate things. Testnet would tell us if it's a client issue or not however. Did you verify the checksum?

**Comment by wowology** [comment]
what is the checksum?

**Comment by wowology** [comment]
I'm running on MAC

**Comment by stevesbrain** [comment]
@wowology The checksum verifies that the file downloaded correctly. See this release page for more details (the hashes are what you want to compare):

https://github.com/CityOfZion/neon-wallet/releases/tag/0.0.7

**Comment by wowology** [comment]
yes, I downloaded the 7 version from this site, so I should have the correct file for wallet.  What else can I do?

**Comment by wowology** [comment]
I have downloaded the wallet a second time from the page, so that should be the correct one.  But still does not work.  Do I need to go to version 5, or what do I do?

**Comment by wowology** [comment]
I'm able to send NEO, but not GAS

**Comment by wowology** [comment]
just sent one NEO and that arrived on another account

**Comment by wowology** [comment]
What do you recommend I do to get hold of my GAS and be able to transfer it

**Comment by stevesbrain** [comment]
@wowology I'm not suggesting you download the file again, I was simply suggesting you check the hash using the command on that page. Does the hash match?

Feel free to try version 0.0.5 - some people have had success with this.

**Comment by wowology** [comment]
Hi Steve,

I've downloaded the version 5 and with that version I can send GAS.  So please have that issue fixed in the version 8.

Thanks for your support!

**Comment by stevesbrain** [comment]
@wowology I'm not actually involved with this project at all, so I can't give you any guarantee it will be fixed in version 8 - I'm just a user like you :) Happy it's all sorted though!

**Comment by anastrophe** [comment]
I realize this is closed, but wanted to add something I discovered. In the 0.0.7 wallet, I was experiencing exactly the same behavior trying to send gas - always 'transaction failed'. So I took the advise and reinstalled 0.0.5, and tried again. Same thing. Then it occurred to me that may it was input parsing. At the time, I had 0.6998 gas. I was trying to transfer just the .6. So I was entering ".6". Failed. Then I tried "0.6". Failed. Then I tried "0.6000" - transaction processed. I reinstalled version 0.0.7, and tried the same with the remainder - as long as I entered it *exactly* as the amount was displayed, the transaction went through fine.

Sorry for the length of this, but this seems to be the problem - the interface insists that you enter your amounts in exact decimal matching how your gas balance is displayed. ".6" or "0.6" are not the same as "0.6000" when trying to initiate a transfer.
